cmd/prometheus/main.go

```diff
@@ -343,7 +343,7 @@ func main() {
 	a.Flag("web.page-title", "Document title of Prometheus instance.").
 		Default("Prometheus Time Series Collection and Processing Server").StringVar(&cfg.web.PageTitle)
 
-	a.Flag("web.cors.origin", `Regex for CORS origin. It is fully anchored. Example: 'https?://(domain1|domain2)\.com'`).
+	a.Flag("web.cors.origin", `Regex for CORS origin. It is fully anchored. Example: 'https?://(domain1\|domain2)\.com'`).
```

Comment:
This is just supposed to fix the documentation Markdown, AFAICT. This also changes Prometheus' help output though, in that the example regexp changes (so that it doesn't recognize "domain1 or "domain2" any more, but instead1 "domain1|domain2"). Shouldn't we instead try to fix the Markdown generation, so it escapes the `|` character?

Reply:
Thanks for responding!
Yes, I'm trying to only fix the documentation Markdown, but the test CI won't pass successfully.
What else file should I change to make the test pass?

Reply:
I've committed new changes. PTAL.

Reply:
Looks like the right solution to me. Trying to figure out how to render the documentation, to see the effect of your fix.

Reply:
[preview](https://github.com/B1F030/prometheus/blob/patch-1/docs/command-line/prometheus.md#:~:text=domain1) in my branch

Reply:
Oh yeah good point, you can just preview the Markdown in GitHub.